chia/util/default_root.py

```diff
@@ -2,3 +2,5 @@
 from pathlib import Path
 
 DEFAULT_ROOT_PATH = Path(os.path.expanduser(os.getenv("CHIA_ROOT", "~/.chia/mainnet"))).resolve()
+
+DEFAULT_KEYS_ROOT_PATH = Path(os.path.expanduser(os.getenv("CHIA_KEYS_ROOT", "~/.chia_keys"))).resolve()
```

Comment:
Why allow overriding instead of using the --keys-root CLI argument? is it for GUI support?

Reply:
No specific reason other than I was following the pattern for DEFAULT_ROOT_PATH to be consistent. Using an env var instead of --keys-root-path would be my personal preference if I had to use the CLI for several operations, but if there's a reason to avoid this I'm fine with removing it.

Reply:
Hmm Ideally we want to have only one way to do things instead of two. That said, we are pretty deep into the CHIA_ROOT rabbit hole so maybe it's fine to do it the same way for CHIA_KEYS_ROOT, and slowly migrate over to the -- way of doing things